src/main/app/forms/models/address.ts

```diff
@@ -62,7 +63,10 @@ export class Address implements CompletableTask {
   postcode?: string
 
   @ValidateIf(o => !o.addressVisible && !o.addressSelectorVisible, { groups: ['claimant', 'defendant', 'response'] })
-  @IsNotBlank({ message: ValidationErrors.POSTCODE_REQUIRED, groups: ['claimant', 'defendant', 'response'] })
+  @AllFieldsArePopulated(['postcode', 'line1'], {
```

Comment:
These extra validation rules required by postcode lookup are so unreadable :(
  
It is so complex that at least we should have proper unit tests to describe the rules if we want to accept that tech debt.

Reply:
Why didn't you include 'city' property as well? I cannot find reason for having just two of three required.

Reply:
I agree that it's quite unintuitive as it is now. What do you think about renaming the annotation? `@ExtraFormFieldsArePopulated`? `@SupplementaryFormFieldsArePopulated`?